systemdev/mms-infrastructure.rst

```diff
@@ -0,0 +1,82 @@
+MMS infrastructure components:
+==============================
+
+
+**oFono [1]** - responsible for providing the data context used to transfer mms data (image/music). It also propagates wap push notifications to upper layers.
+
+**nuntium [2]** - daemon that listens to wap push notifications and activates the mms data context on ofono on demand to send/receive mms's. It provides a local store, which will probably not be used by ubuntu-touch. The history service component will be used instead.
+
+**telepathy-ofono [3]**  - talks to nuntium through dbus and will be used to both relay messages from phone-app to nuntium and inject into the telepathy infrastructure mms's received by nuntium as multi-part messages [4]. This component also needs to mark messages as read and delete successfully received messages from nuntium.
```

Comment:
"will be" again ... do you know whether it *is* used currently?

Reply:
i removed it, as it is used finally :)